integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.client.integration;
+
+import akka.actor.ActorSystem;
+import akka.japi.Effect;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import com.lightbend.lagom.internal.client.*;
+import com.lightbend.lagom.internal.registry.ServiceRegistry;
+import com.lightbend.lagom.internal.registry.ServiceRegistryServiceLocator;
+import com.lightbend.lagom.javadsl.api.ServiceInfo;
+import com.lightbend.lagom.javadsl.api.ServiceLocator;
+import com.lightbend.lagom.javadsl.jackson.JacksonExceptionSerializer;
+import com.lightbend.lagom.javadsl.jackson.JacksonSerializerFactory;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import org.pcollections.PVector;
+import org.pcollections.TreePVector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.api.Configuration;
+import play.api.Environment;
+import play.api.Mode;
+import play.api.inject.ApplicationLifecycle;
+import play.api.libs.ws.WSClient;
+import play.api.libs.ws.WSClientConfig;
+import play.api.libs.ws.WSConfigParser;
+import play.api.libs.ws.ahc.AhcConfigBuilder;
+import play.api.libs.ws.ahc.AhcWSClient;
+import play.api.libs.ws.ahc.AhcWSClientConfig;
+import play.api.libs.ws.ahc.AhcWSClientConfigParser;
+import play.api.libs.ws.ssl.SystemConfiguration;
+import scala.Function0;
+import scala.concurrent.Future;
+
+import java.io.Closeable;
+import java.io.File;
+import java.net.URI;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+
+/**
+ * Factory for creating Lagom service clients.
+ *
+ * This is designed for use from non Lagom systems.
+ *
+ * Generally, there should be only one instance of this per system.  Additionally, since it holds thread and connection
```

Comment:
Why not enforcing this constraint and making it impossible to create more than one instance? Or do you already foreseen a use case for having more than one instance?